quickhash.c

```diff
@@ -207,6 +214,7 @@ int php_qh_save_chars_to_string_func(void *context, char *buffer, uint32_t eleme
 
 int32_t php_qh_get_size_from_stream(void *context)
 {
+	TSRMLS_FETCH();
```

Comment:
How often is this called? If it's more than once per stream opening, we might want to look at a different way for doing this.